packages/react-magma-dom/src/components/TreeView/TreeView.stories.tsx

```diff
@@ -2384,3 +2405,305 @@ ComplexWithTopLevelNotSelectable.args = {
+        justify={FlexJustify.spaceBetween}
+        wrap={FlexWrap.wrap}
+      >
+        <Flex behavior={FlexBehavior.item}>
+          <Hyperlink to="google.com" target="_blank" hasUnderline={false}>
+            Most common activity length is 39 chars but what if longer
+          </Hyperlink>
+        </Flex>
+        <Flex
+          behavior={FlexBehavior.item}
+          style={{ marginLeft: '40px' }}
+          justify={FlexJustify.flexEnd}
+          alignContent={FlexAlignContent.flexEnd}
+        >
+          <Flex
+            behavior={FlexBehavior.container}
+            style={{ gap: '8px' }}
+            wrap={FlexWrap.nowrap}
+          >
+            <Flex behavior={FlexBehavior.item}>
+              <Tag size={TagSize.small} icon={<LockIcon />}>
+                Hidden from Students
+              </Tag>
+            </Flex>
+            <Flex behavior={FlexBehavior.item}>
+              <Dropdown>
+                <DropdownButton
+                  aria-label="Extra icon example"
+                  color={ButtonColor.secondary}
+                  size={ButtonSize.small}
+                  icon={<MoreHorizIcon />}
+                  shape={ButtonShape.fill}
+                />
+                <DropdownContent>
+                  <DropdownMenuItem icon={<EditIcon aria-hidden />}>
+                    Rename
+                  </DropdownMenuItem>
+                </DropdownContent>
+              </Dropdown>
+            </Flex>
+          </Flex>
+        </Flex>
+      </Flex>
+    );
+  };
+
+  const folderLabel = (label: any) => {
+    return (
+      <Flex
+        behavior={FlexBehavior.container}
+        justify={FlexJustify.spaceBetween}
+        wrap={FlexWrap.nowrap}
+      >
+        <Flex behavior={FlexBehavior.item}>{label}</Flex>
+        <Flex
+          behavior={FlexBehavior.item}
+          style={{ display: 'inline-flex', marginLeft: '40px' }}
+        >
+          <Paragraph
+            visualStyle={TypographyVisualStyle.bodySmall}
+            noMargins
+            color={TypographyColor.subdued}
+            style={{ marginRight: magma.spaceScale.spacing03 }}
+          />
+          <Dropdown>
+            <DropdownButton
+              aria-label="Extra icon example"
+              color={ButtonColor.secondary}
+              size={ButtonSize.small}
+              icon={<MoreHorizIcon />}
+              shape={ButtonShape.fill}
+            />
+            <DropdownContent>
+              <DropdownMenuItem icon={<EditIcon aria-hidden />}>
+                Rename
+              </DropdownMenuItem>
+            </DropdownContent>
+          </Dropdown>
+        </Flex>
+      </Flex>
+    );
+  };
+
+  const additionalContent = () => {
+    return (
+      <div style={{ width: '100%' }}>
+        <Paragraph noTopMargin visualStyle={TypographyVisualStyle.bodyXSmall}>
+          Due: xx/xx/xxx · Submitted: 12 · Missing: 3
+        </Paragraph>
+        <ButtonGroup style={{ marginBottom: '16px' }}>
```

Comment:
We can remove this spacing since now it's built into the `AdditionalContentWrapper`